keps/sig-scheduling/20200114-taint-based-evictions.md

```diff
@@ -0,0 +1,98 @@
+---
+title: Promote Taint Based Evictions to GA
+authors:
+  - "@damemi"
+owning-sig: sig-scheduling
+participating-sigs:
+  - sig-scheduling
+reviewers:
+  - "@Huang-Wei"
+approvers:
+  - "@liggitt"
+editor:
+creation-date: 2020-01-14
+last-updated: 2020-01-14
+status: implementable
+see-also:
+replaces:
+superseded-by:
+---
+
+# Promote Taint Based Evictions to GA
+
+## Table of Contents
+
+<!-- toc -->
+- [Summary](#summary)
+- [Motivation](#motivation)
+  - [Goals](#goals)
+  - [Non-Goals](#non-goals)
+- [Proposal](#proposal)
+  - [Risks and Mitigations](#risks-and-mitigations)
+- [Graduation Criteria](#graduation-criteria)
+- [Testing Plan](#testing-plan)
+  - [Unit tests](#unit-tests)
+  - [Integration tests](#integration-tests)
+  - [e2e tests](#e2e-tests)
+- [Implementation History](#implementation-history)
+<!-- /toc -->
+
+## Summary
+
+Taint Based Evictions was introduced as an alpha feature in Kubernetes 1.6 and was promoted to
+beta in 1.13. Taint Based Evictions evicts pods from a node based on taints applied to the node
+representing node conditions. By setting `tolerationSeconds`, users can specify how long a pod
+should stay bound to a node tainted with one of these conditions.
+
+## Motivation
+
+Taint Based Evictions has been in beta since 1.13 and has functioned well since then.
+
+### Goals
+
+Promote Taint Based Evictions to GA.
```

Comment:
We should say "Ensure nodes are tainted properly with a NoExecute effect when it's not ready or unreachable, so that scheduler can use taints to make scheduling decisions consistently."